controller/controller.go

```diff
@@ -216,3 +216,10 @@ func (c *Impl) processNextWorkItem() bool {
 
 	return true
 }
+
+// GlobalResync enqueues all objects from the passed SharedInformer
+func (c *Impl) GlobalResync(si cache.SharedInformer) {
+	for _, key := range si.GetStore().ListKeys() {
+		c.EnqueueKey(key)
```

Comment:
Copying the comment I had for the previous iteration: List() returns `[]interface{}` - `c.Enqueue` expects a "namespace/name" string. You might need to pass in a func to this method to get the key from the object and enqueue that instead.

Reply:
My mistake. This is doing the right thing. Sorry for the confusion.